lib/open_food_network/variant_stock.rb

```diff
@@ -1,19 +1,31 @@
 require 'active_support/concern'
 
-# These methods were available in Spree 1, but were removed in Spree 2.
-# We would still like to use them. Therefore we use only a single stock location
-# (default stock location) and use it to track the `count_on_hand` value that
-# was previously a database column on variants.
+# These methods were available in Spree 1, but were removed in Spree 2.  We
+# would still like to use them so that we still give support to the consumers
+# of this methods, making the upgrade backward compatible.
+#
+# Therefore we use only a single stock item per variant, which is associated to
+# a single stock location per instance (default stock location) and use it to
+# track the `count_on_hand` value that was previously a database column on
+# variants. See
+# https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-Upgrade%3A-Stock-locations
+# for details.
 #
 # We may decide to deprecate these methods after we designed the Network feature.
```

Comment:
We may have the deprecation warning here, for example, "Some methods are or may become deprecated." but I dont think we should mention roadmap/future items in code comments ever. I'd remove the reference to Network feature.